docs/misc/writing_docs.md

```diff
@@ -0,0 +1,351 @@
+
+The goal of documentation is to provide information to the reader. This means that, aside from
+correctness, the most important metrics of documentation are its efficiency (words per idea) and
+efficacy (readability of words and retention of ideas).
+
+Reading a sentence in isolation helps to analyze the readability of an idea in a narrow context,
+but reading multiple sentences (or a paragraph) at a time can be a good signal of how well
+information flows.
+
+## Tip #13: Be pedantic about grammar.
+
+English grammar can often be flexible, subjectively readable, or confusing (there is more than one
+way to skin a cat). Being clear and using [simpler, well-structured sentences][concise-sentences]
+can [help with readability][readability] by presenting digestible, unambiguous ideas.
+[Verb tense][verb-tense] and [pronouns][pronouns] can play an important role as well. Ultimately,
+the best litmus test may be the simplest: whether a reader needs to read a sentence twice to
+understand it (technical terms notwithstanding).
+
+Realistically, it is impractical to strive for or expect perfect grammar. The goal isn't to write
+flawless English; the goal is to write _clear, concise, unambiguous, and readable_ documentation,
+for which grammar is an important (albeit not the only) tool.
+
+## Tip 14: Use verb tenses consistently.
+
+Generally speaking, the bulk of documentation only makes sense when written in the present tense
+(its "common tense", so to speak), as objects are typically referred to in the abstract (e.g.
+"Component X" is abstract, "The running instance of Component X" proverbially concrete) and do not
+usually involve time (e.g. "Component X is an upstream of Component Y" refers to nominal abstract
+state, "Component X will be an upstream of Component Y" is prospective state). Of course, this is
+not always true: documentation can tell a story, and tenses should be used as needed to express ideas
+relative to time.
+
+However, cases of general fact, such as describing cause and effect:
+
+    Value X -> Behavior Y
+
+should be described in the present tense:
+
+    Setting Value X causes Behavior Y.
+
+versus the future tense:
+
+    Setting Value X will cause Behavior Y.
+
+in order to be consistent with the documentation's common tense, to maintain a time-less (and thus
+ongoing) objective truth, and to contrast actual time-relative statements (e.g. "Setting Value X
+causes Behavior Y, but will be updated to cause Behavior Z in the future").
+
+## Tip #15: Make consistent references.
+
+When communicating verbally, there is no need to distinguish between represented forms for a given
+artifact, value, method, etc - one can simply say "foo bar" to reference "the member function or
+property `Bar` on type `Foo`", or "foo dot bar" to reference "the YAML key `bar` nested under
+the key `foo`" (as simple examples).
+
+In written documentation, however, these references can either maintain continuity throughout a
```

Comment:
This is a good point! I agree that there's not a consistent style on this overall (sadly), but should we recommend a standard for the purposes of this repo (as opposed to potentially having different styles on different pages)? Maybe not worth it.

Reply:
added some more detail around this, let me know what you think